src/Symfony/Component/Translation/Command/TranslationPullCommand.php

```diff
@@ -142,6 +144,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             'path' => end($this->transPaths),
             'xliff_version' => $xliffVersion,
             'default_locale' => $this->defaultLocale,
+            'as_tree' => $asTree,
```

Comment:
Looking at the YamlDumper, it seams like we only care for boolean value

https://github.com/symfony/symfony/blob/015d5015e353ee5448bf7c350de0db4a03f8e13a/src/Symfony/Component/Translation/Dumper/YamlFileDumper.php#L41

Reply:
Thank you. It is now casted to a boolean value.